packages/tenants/src/context/__tests__/TenantAgentContextProvider.test.ts

```diff
@@ -62,6 +63,11 @@ describe('TenantAgentContextProvider', () => {
 
       const tenantAgentContext = jest.fn() as unknown as AgentContext
 
+      container.registerInstance(CacheModuleConfig, {
+        cache: new CacheModule({
+          cache: new InMemoryLruCache({ limit: 100 }),
+        }).config.cache as unknown as CacheModule,
+      })
```

Comment:
I think you can simplify this

Does this work?

```suggestion
      container.registerInstance(CacheModuleConfig, new CacheModuleConfig({
        cache: new InMemoryLruCache({ limit: 100 })
       })
```

Reply:
Yes, This test case is working I have tested it on my local machine.